theindividualcompany/cal.com
# SW-115: Clean up 404 page

## What does this PR do?

> 
> 
> 
> Fixes # (issue)
> 
> 
> 
> ## Requirement/Documentation
> 
> 
> 
> - If there is a requirement document, please, share it here.
> - If there is ab UI/UX design document, please, share it here.
> 
> ## Type of change
> 
> 
> 
> - [ ] Bug fix (non-breaking change which fixes an issue)
> - [ ] Chore (refactoring code, technical debt, workflow improvements)
> - [ ] New feature (non-breaking change which adds functionality)
> - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
> - [ ] This change requires a documentation update
> 
> ## How should this be tested?
> 
> 
> 
> - Are there environment variables that should be set?
> - What are the minimal test data to have?
> - What is expected (happy path) to have (input and output)?
> - Any other important info that could help to test that PR
> 
> ## Mandatory Tasks
> 
> - [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.
> 
> ## Checklist
> 
> 
> 
> - I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
> - My code doesn't follow the style guidelines of this project
> - I haven't commented my code, particularly in hard-to-understand areas
> - I haven't checked if my PR needs changes to the documentation
> - I haven't checked if my changes generate no new warnings
> - I haven't added tests that prove my fix is effective or that my feature works
> - I haven't checked if new and existing unit tests pass locally with my changes
>
